apps/heft/src/cli/HeftToolsCommandLineParser.ts

```diff
@@ -115,6 +119,66 @@ export class HeftToolsCommandLineParser extends CommandLineParser {
       registerAction: <TParameters>(options: ICustomActionOptions<TParameters>) => {
         const action: CustomAction<TParameters> = new CustomAction(options, actionOptions);
         this.addAction(action);
+      },
+      registerParameters: <TParameters, TAction extends HeftActionBase>(
```

Comment:
This should throw if it's called after the parameters have been parsed.

Reply:
Fixed.